packages/ckeditor5-autosave/src/autosave.js

```diff
@@ -222,39 +247,91 @@ export default class Autosave extends Plugin {
 	 * It waits for the result and then removes the created pending action.
 	 *
 	 * @private
+	 * @param {Boolean} [manualSave=false] Whether the save was triggered by another plugin (`true`) or by the autosave plugin, after
+	 * the model has changed (`false`).
+	 * @returns {Promise} A promise that will be resolved when the autosave callback is finished.
 	 */
-	_save() {
+	_save( manualSave = false ) {
+		if ( this._savePromise ) {
+			if ( manualSave ) {
```

Comment:
There is a missing else case coverage.

![](https://user-images.githubusercontent.com/1232187/148815037-5f696c0c-97c2-473c-be4d-a573353b3bdd.png)

Reply:
Sorry for that, I have problems with CC on my local machine for some reason and had to check it on CI.